api/src/pcapi/connectors/cine_digital_service.py

```diff
@@ -54,7 +54,7 @@ def get_resource(
         response.raise_for_status()
 
     except requests.exceptions.RequestException as e:
-        raise cds_exceptions.CineDigitalServiceAPIException(f"API CDS - url : {url} - error : {e}")
+        raise cds_exceptions.CineDigitalServiceAPIException(f"API CDS - error : {e}")
```

Comment:
(détail / suggestion) On pourrait préciser le message d'erreur en indiquant le type de ressource et l'action qui a échoué. Quelque chose comme : 

```suggestion
        raise cds_exceptions.CineDigitalServiceAPIException(f"Error on CDS API while getting {resource}: {e}")
```

... ou bien une autre formulation : 

```suggestion
        raise cds_exceptions.CineDigitalServiceAPIException(f"Error on CDS API on GET {resource}: {e}")
```


Idem dans les autres fonction avec les variantes "while putting..." et "while posting".

Qu'en penses-tu ?